drivers/storage/class/cdrom/cdrom.inf

```diff
@@ -57,6 +57,12 @@ CDROMClassName = "DVD/CD-ROM drives"
 GenCdRom.DeviceDesc = "CD-ROM Drive"
 GenericMfg = "(Generic DVD/CD-ROM Drives)"
 
+[Strings.040C]
```

Comment:
Can you keep the sections sorted please? Ignore the '04', then it goes 08,09,0a,0b,0c, ...
And in doing that, you'd see there is already a .040c section.

Reply:
ok